core/src/main/java/org/jdbi/v3/core/statement/SqlStatement.java

```diff
@@ -183,6 +184,33 @@ public This bindBean(String prefix, Object bean) {
         return bindNamedArgumentFinder(new BeanPropertyArguments(prefix, bean));
     }
 
+    /**
+     * Binds named parameters from object properties on the argument.
+     *
+     * @param pojo source of named parameter values to use as arguments
+     *
+     * @return modified statement
+     */
+    @Beta
+    public This bindProperties(Object pojo) {
```

Comment:
What do you think about `bindPojo()` as a more concise name?

Reply:
This needs to be documented that you can only use this method if you've registered a property-based mapper